.jenkins/jenkins.sh

```diff
@@ -75,7 +76,9 @@ echo "PYTHON env ${python_env}"
 # If the backend is a GTC backend we fetch the caches
 if [[ $backend != *numpy* ]];then
     echo "Fetching for exisintg gt_caches"
-    . ${JENKINS_DIR}/actions/fetch_caches.sh $backend $experiment
+    cd ${PACE_DIR}/fv3gfs-physics
+    ${JENKINS_DIR}/fetch_caches.sh $backend $experiment
+    cd ${PACE_DIR}
```

Comment:
Not really something you can fix right now/in this PR, so no changes needed: I want to point out this (pre-existing code) hard-codes that every test whose backend doesn't contain numpy using jenkins.sh should fetch caches in the physics directory. For example, if we moved the pace-util tests to use this jenkins.sh, the backend would be an empty string, and caches would get fetched for an empty string backend.

It would be nice if we could run code to fetch caches only for the tests which require fetching caches. I think doing this would require adding a similar concept to "actions" except that they're scripts which are never submitted to a slurm job/docker container/etc., they always run bare-metal, and basically act as hot-swappable patches of code that you would put in jenkins.sh otherwise.